src/components/ForestTypeComparison/lu/ForestTypeTab.js

```diff
@@ -2,78 +2,86 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Table } from 'semantic-ui-react';
-import { useDispatch } from 'react-redux';
+import useIsMobile from '../../../hooks/useIsMobile';
 
+import ComparisonCell from './ComparisonCell';
+import ForestTypeLink from './ForestTypeLink';
 import { forestTypeMapping } from '../../ForestTypeDescription/lu/utils';
-import {
-  setForestTypeDescription,
-  setForestTypeModal,
-} from '../../../store/actions';
-
+import { getHasSameValues } from './utils';
 import comparisonStyles from '../ForestTypeComparison.module.css';
-import descriptionStyles from '../../ForestTypeDescription/ForestTypeDescription.module.css';
-
-const getPrecentageString = (arr) =>
-  arr.every((val) => val !== null && val !== undefined)
-    ? `${arr.join('-')}%`
-    : '-';
-
-const getValueIsSame = (arr1, arr2) =>
-  getPrecentageString(arr1) === getPrecentageString(arr2);
-
-const getHasSameValues = (currentInfo, infoArray, field) =>
-  infoArray.some(
-    (ft) =>
-      currentInfo.code !== ft.code &&
-      getValueIsSame(currentInfo[field], ft[field]),
-  );
-
-const Cell = ({ data, hasSameValues }) => (
-  <Table.Cell
-    className={hasSameValues ? comparisonStyles.comparisonIsSame : undefined}
-  >
-    {getPrecentageString(data)}
-  </Table.Cell>
-);
 
-Cell.propTypes = {
-  data: PropTypes.arrayOf(PropTypes.number).isRequired,
-  hasSameValues: PropTypes.bool.isRequired,
+const getPioneerTreeTypes = (info, allInfos) => {
+  const allTreeTypes = allInfos.reduce((pioneerTypes, current) => {
+    const currentPioneersTypes = current.pioneerTreeTypes
+      ?.replace(' ', '')
```

Comment:
Maybe we should use an array for `pioneerTreeTypes` in `tree-lib`?

Reply:
Done